pkg/surveyext/editor.go

```diff
@@ -59,7 +59,9 @@ var EditorQuestionTemplate = `
 {{- else }}
   {{- if and .Help (not .ShowHelp)}}{{color "cyan"}}[{{ .Config.HelpInput }} for help]{{color "reset"}} {{end}}
   {{- if and .Default (not .HideDefault)}}{{color "white"}}({{.Default}}) {{color "reset"}}{{end}}
-	{{- color "cyan"}}[(e) to launch {{ .EditorCommand }}{{- if .BlankAllowed }}, enter to skip{{ end }}] {{color "reset"}}
+	{{- color "cyan"}}[(e) to launch {{ .EditorCommand }}
+	{{- if .BlankAllowed }}, (enter/return) to keep empty{{ end }}
+	{{- if .Default }}, (d) to accept default{{ end }}] {{color "reset"}}
```

Comment:
Nice approach! If there is a template, the editor prompt will now read:

```
[(e) to launch editor, (enter/return) to keep empty, (d) to accept default]
```

1. Is there another scenario in which there will be a default value other than there being a template?
2. Will people understand what "default" refers to in this case? Keep in mind that users might not be aware that there is a template in a repository by the time they are opening an issue or PR

Reply:
1. Doesn't look like it. There was an initial option of the "autofilled" values being there as the default ones; I checked it now, but it looks like that is a code branch that is never encountered. So maybe we should even consider removing the `defs defaults` paramenter from the `titleBodySurvey` function itself?  
2. I believe at least people familiar with the repository would be knowing this. Like #1219 .

Reply:
Since we're dealing with multiple options, we should explore exposing them in a list. Does something like this make sense?

```
? Body  [Use arrows to move, type to filter]
> Continue with commit body 
  Continue with blank body 
  Compose in editor 
```

Reply:
I think expanding this into a list is A+. The little editor widget is being asked to do too much, here.